src/auth/auth.action.js

```diff
@@ -26,6 +33,19 @@ export const auth = (code, state) => {
   };
 };
 
+export const signOut = callback => {
```

Comment:
Hey! Is there a way we can omit this callback in this action? It looks like a code smell to me.